homeassistant/core.py

```diff
@@ -1260,7 +1260,7 @@ class Service:
 
     def __init__(
         self,
-        func: Callable,
+        func: Callable[[ServiceCall], None | Awaitable[None]],
```

Comment:
Dependent on the outcome of the discussion in https://github.com/home-assistant/core/pull/63239#discussion_r777303337

I believe I could make sense to create a new TypeAlias for it. It could also be used for the `register` and `async_register` functions.

Reply:
Not going to add an alias for it. https://github.com/home-assistant/core/pull/63239#discussion_r779199332
Please consider it resolved. (Just not marking it as such so that it stays visible.)